.eslintrc.json

```diff
@@ -13,7 +13,12 @@
   ],
   "rules": {
     "react-hooks/exhaustive-deps": "error",
-    "import/no-relative-parent-imports": "error",
+    "import/no-relative-parent-imports": [
+      "error",
+      {
+        "ignore": ["@/"]
```

Comment:
I think it should be src instead of @ here.

https://github.com/visualize-admin/electricity-prices-switzerland/blob/main/tsconfig.json#L19

Reply:
@ptbrowne it looks like it doesn't work either way 😭 I tried with and without different ignore patterns plus added the `import/recommended` extension below:

```json
  "extends": [
    "plugin:@typescript-eslint/recommended",
    "plugin:import/recommended"
  ],
```

but still it doesn't highlight relative imports. Do you have any idea? Otherwise I'll remove the rule, as it doesn't do anything now 🥲